floodwatch/src/js/components/comparisontools.js

```diff
@@ -9,45 +9,51 @@ import type {Preset, Filter, FilterLogic} from './filtertypes'
 import type {VisibilityMap} from './Compare'
 
 export function getVisibilityMap(lD, rD): VisibilityMap {
```

Comment:
This function is actually a great example of `let` and `const` - `visibilityMap` can be `const` because it'll never change, whereas `comparisonData` has to get assigned.